lib/workspace.js

```diff
@@ -393,23 +432,21 @@ module.exports = {
       config.workspaceJobsDirectoryOwnerGid
     );
 
-    if (localPathExists) {
-      debug(`Syncing ${localPath} to ${remotePath}`);
-      await fse.copy(localPath, sourcePath);
-    }
-
-    if (dynamicFiles) {
-      await async.eachSeries(dynamicFiles, async ({ name, localPath, buffer }) => {
-        await fse.ensureDir(path.join(sourcePath, path.dirname(name)));
-        if (localPath) {
-          await fse.copy(localPath, path.join(sourcePath, name));
-        } else {
-          await fse.writeFile(path.join(sourcePath, name), buffer);
+    if (allWorkspaceFiles) {
```

Comment:
I don't think this will do what you want it to; perhaps you want this?

```suggestion
    if (allWorkspaceFiles.length > 0) {
```

Reply:
Yeah, used to Python where an empty array is falsy...